builtin/mainmenu/dlg_settings_advanced.lua

```diff
@@ -517,24 +517,20 @@ end
 
```

Comment:
I'm coming too late but `%g` string serialization needs to be applied in the previous function `get_current_np_group` too

Causes rounding issues on the dialogs
![image](https://user-images.githubusercontent.com/6905002/152597816-0935cef7-23de-4dd1-b81b-381565d01607.png)

(I don't know how to make a comment outside the scope of the changes)